physical_properties/pKa/analysis/analysis_of_typeIII_predictions/README.md

```diff
@@ -3,15 +3,25 @@
 General analysis of typeIII macroscopic pKa predictions include calculated vs predicted pKa correlation plots and 5 performance statistics (RMSE, MAE, ME, R^2 and linear regression slope(m) ) for all the submissions. 
 95%-percentile bootstrap confidence intervals of all the statistics were reported. 
 
-One-to-one matching of predicted pKas to experimental pKas was performed based on minimum absolute error principle.
-Predicted pKas were matched to experimental pKa values that minimize the absolute error.
-When more than one predicted pKa were matched to the same experimental pKa, only the predicted pka that has the lowest absolute error was kept. 
+One-to-one matching of predicted pKas to experimental pKas was performed with two different methods based on minimum error principle:
+1. **Closest**: Each predicted pKa is matched to experimental pKa values that minimize the absolute error of that pair.
+When more than one predicted pKa match to the same experimental pKa, only the predicted pka that has the lowest absolute error is kept. 
 Predicted pKas that were not matched to experimental pKas were excluded from this analysis.
+2. **Hungarian**: Experimental pKas and predicted pKas are matched following Hungarian algorithm. It finds the optimum assignment between experimental and predicted set of pKas that minimizes linear sum of squared errors of all pairwise matches. Cost is defined as squared error instead of absolute error to minimize the effect of extra predicted pKas to overall matching. We acknowledge Kiril Lanevskij for suggesting this alternative assignment of experimental pKas.
```

Comment:
Could you add a reference for the Hungarian algorithm?

Reply:
I will. Thank you.